actions/chats.js

```diff
@@ -0,0 +1,76 @@
+import * as types from '../constants/actionTypes';
+import { baseUrl } from '../constants/api';
+import {
+    handleResponse,
+    storeData,
+    addAuthHeader
+} from '../helpers/api';
+
+export const setCurrentChat = (chat) => {
```

Comment:
if you're setting something in the store that doesn't need to call the api, you can do it like this